src/widgetastic_patternfly/__init__.py

```diff
@@ -593,23 +593,40 @@ class Tab(View, ClickableMixin):
 
     Selects itself automatically when any child widget gets accessed, ensuring that the widget is
     visible.
+
+    Args:
+        TAB_NAME: Name of the tab
+        IGNORE_WHEN_NOT_PRESENT: If hte tabs are not present, do not bother us with exceptions.
     """
     TAB_NAME = None
     INDIRECT = True
+    IGNORE_WHEN_NOT_PRESENT = False
     ROOT = ParametrizedLocator(
         './/ul[contains(@class, "nav-tabs")]/li[normalize-space(.)={@tab_name|quote}]')
 
     @property
     def tab_name(self):
         return self.TAB_NAME or type(self).__name__.capitalize()
 
+    @property
+    def present(self):
+        try:
+            self.is_active
+        except NoSuchElementException:
+            return False
+        else:
+            return True
+
     def is_active(self):
         return 'active' in self.browser.classes(self)
 
     def is_disabled(self):
         return 'disabled' in self.browser.classes(self)
 
     def select(self):
+        if self.IGNORE_WHEN_NOT_PRESENT:
```

Comment:
at first glance this gives an feeling of allowing code to easily look working

for better understanding this should come with a clear note and a example at least

Reply:
No, it will not. This is to ease the upgrade if old version did not have tabs and new has tabs or vice versa. Remember that by default, widgetastic checks for visibility, so the things "contained" in a tab will yell anyway if something is wrong :)

Reply:
What's the difference between `present` and `is_displayed`?

Reply:
Right. My brain processes after weekend. I will remove present :)